src/main/python/keycodes/keycodes.py

```diff
@@ -326,10 +326,12 @@ def resolve(cls, qmk_constant):
 KEYCODES_ISO.extend(KEYCODES_ISO_KR)
 
 KEYCODES_LAYERS = []
-RESET_KEYCODE = "RESET"
+RESET_KEYCODE = "QK_BOOT"
 
 KEYCODES_BOOT = [
-    K("RESET", "Reset", "Reboot to bootloader")
+    K("QK_BOOT", "Boot-\nloader", "Put the keyboard into bootloader mode for flashing"),
```

Comment:
it should have an alias to the original "RESET" or old exported *.vil wouldn't load anymore